airbyte-integrations/connectors/destination-s3/src/main/resources/spec.json

```diff
@@ -252,14 +252,21 @@
           },
           {
             "title": "JSON Lines: Newline-delimited JSON",
-            "required": ["format_type"],
+            "required": ["format_type", "flattening"],
```

Comment:
To be safe, I would want you to test upgrade/downgrade scenarios:
 - Create a connection with current version of this connector
 - Run initial sync
 - Run an incremental/append sync
 - Upgrade connector
 - Run an incremental/append sync - validate that the output is in the same format as before the upgrade
 - Downgrade connector
 - Run an incremental/append sync

there should be no failures in any of the syncs that you are running in the sequence above w/o resets.